src/sagas.js

```diff
@@ -295,6 +321,12 @@ function* schedulerPerMinute (action) {
   }
 }
 
+let sagasFunctions = {
```

Comment:
Please add comment describing purpose of this wrapping object.

Something like:
Saga functions to be called from subsagas .

Or better even more descriptive.